Lib/test/test_ast/test_ast.py

```diff
@@ -84,6 +84,23 @@ def test_AST_objects(self):
             # "ast.AST constructor takes 0 positional arguments"
             ast.AST(2)
 
+    def test_AST_fields_NULL_check(self):
+        # See: https://github.com/python/cpython/issues/126105
+        old_value = ast.AST._fields
```

Comment:
```suggestion
        # See gh-126105
        old_value = ast.AST._fields
```
A little style nit, feel free to ignore.